docker-compose.yml

```diff
@@ -2,13 +2,13 @@
 version: "3"
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper
+    image: https://quay.io/repository/cloudservices/cp-zookeeper
```

Comment:
Technically, there is no http(s) schema. It's `docker://`. Also, these image urls should not include `/repository/`. Did you test this locally?

Reply:
ah ! you're right - sorry , thanks for fixing this :)